planetarypraxis/smartforests
# Display autogenerated map thumbnail and place name for pages

Closes #184 

- Adds a map thumbnail to the metadata sidebar of each page, if it has coordinates. This comes from Mapbox.
- Also auto-generates a place label for coordinates, if one is not set, using Open Street Map.

The logic for these features comes from Airsift. https://github.com/citizensense/airsift3/blob/ce24f9f7ecb78568ab766935029eb11cc34f6b4b/airsift/data/models.py#L132-L145

## How Can It Be Tested?

- Edit a page to have coordinates, save the page and visit the live page. You should see the map and coordinates.

## Screenshots (if appropriate):

See #184 for latest screenshots.

## Types of changes


## Checklist:
